enterprise/backend/test/metabase_enterprise/public_settings_test.clj

```diff
@@ -15,3 +19,13 @@
     (public-settings/enable-password-login! false)
     (is (= false
            (public-settings/enable-password-login)))))
+
+(deftest settings-managers-can-have-uploads-db-access-revoked
```

Comment:
These seems like an odd location for this test. I think it would fit better in a test namespace related to uploads, or in `metabase-enterprise.advanced-permissions.common-test` which is a bit of a catch-all test namespace for EE perms